cdf-fabric-replicator-chart/values.yaml

```diff
@@ -0,0 +1,68 @@
+# Default values for cdf-fabric-replicator-chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository:
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "1.0.0"
+
+replicaCount: 1
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
```

Comment:
what's a secure default value here?

Reply:
Addressed  this along with Security Context here: 

https://github.com/cognitedata/cdf-fabric-replicator/pull/84#discussion_r1586504884